runtime/altair/src/weights/pallet_uniques.rs

```diff
@@ -162,4 +162,19 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(2 as Weight))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
+	fn buy_item() -> Weight {
```

Comment:
I need help here @branan @mustermeiszer to figure out what the weights are or how I find/calculate... them!